Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -1434,6 +1434,51 @@
         ],
         "era" : "run3_GEM",
     },
+    "M2" : {
+        1 : [
+            'Geometry/MuonCommonData/data/mbCommon/2021/v2/mbCommon.xml',
+            'Geometry/MuonCommonData/data/mb1/2021/v1/mb1.xml',
+            'Geometry/MuonCommonData/data/mb2/2021/v1/mb2.xml',
+            'Geometry/MuonCommonData/data/mb3/2021/v1/mb3.xml',
+            'Geometry/MuonCommonData/data/mb4/2015/v2/mb4.xml',
+            'Geometry/MuonCommonData/data/mb4Shield/2021/v1/mb4Shield.xml',
+            'Geometry/MuonCommonData/data/muonYoke/2021/v5/muonYoke.xml',
+            'Geometry/MuonCommonData/data/mf/2021/v3/mf.xml',
+            'Geometry/MuonCommonData/data/rpcf/2021/v1/rpcf.xml',
+            'Geometry/MuonCommonData/data/gemf/TDR_BaseLine/gemf.xml',
+            'Geometry/MuonCommonData/data/gem11/TDR_BaseLine/gem11.xml',
+            'Geometry/MuonCommonData/data/gem21/2023/v1/gem21.xml',
```

Comment:
@bsunanda , is it correct that dict2021Geometry includes 2023 gem entry?

Reply:
Strictly speaking, we have 2 dictionaries one for Run3 and one for Run4. They are names 2021 and 2026 - maybe we can change the names later. I shall ask Kevin about this. It is he who introduced this

Reply:
In the next technical stop the geometry will changed again. May be correct to keep dictionary for 2021 unchanged and create 2023? I mean that if we keep updating the dictionary would we need to to use it for 2022 geometry? Likely no....

Reply:
We can always add a new version in the dictionary. We can always define a new scenario with a suitable combination. That is why the dictionary is good. I doubt if things will change during any technical stop. It will change most likely in the next YETS for 2024. That can be easily accommodated in this scheme.